Hast.Abstractions/Hast.Vitis.Abstractions/Docs/Zynq-PetaLinux.md

```diff
@@ -0,0 +1,169 @@
+ # Building PetaLinux 2020.2 SD card image for Hastlayer (Trenz TE0715-04-30-1C module)

```

Comment:
Ez még WIP, ugye?

Reply:
Igen. Most repó szempontjából a "Preparing the SD card" előtti sornál tartok, ahol a `petalinux-package` van. Azzal akadtam el amit írtam Basecampen. Az előtte levő rész formázás és hozzáadott tartalom szempontjából "kész" a hiányzó _bit_ fájl problémáját leszámítva.

Reply:
That's a _bit_ of an issue.